ci-operator/step-registry/edge-nodes/machineset-manifests/outpost/edge-nodes-machineset-manifests-outpost-ref.yaml

```diff
@@ -0,0 +1,31 @@
+ref:
+  as: edge-nodes-machineset-manifests-outpost
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  grace_period: 10m
+  commands: edge-nodes-machineset-manifests-outpost-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: LOCALZONE_WORKER_NUMBER
```

Comment:
Can we update these env names, only keep the necessary env for outpost case, and replace LOCALZONE to OUTPOST keyword?

Reply:
I'd like to consolidate all LZ/WLZ/Outpost names to *edge-node* after https://github.com/openshift/release/pull/47045 gets merged, they would be easy update them at the same time, WDYT?

Reply:
I am thinking now we are making a dedicated step for outpost, which can be treated as a total separate step with LZ/WLZ case, and we are unsure when https://github.com/openshift/release/pull/47045 can be merged, I guess it is a chance to do some enhance in advance.

Reply:
Do you mean Outpost is totally separated from edge node? Edge nodes only contain LZ and WLZ?

Reply:
I meant the outpost folder is still placed in edge-nodes, but the ENV var name, we can remove all the LZ/WLZ keywords.

Reply:
Okay, rename all vars in outpost to `edge_node*` in this PR.